samples/docker-domain/README.md

```diff
@@ -28,7 +28,9 @@ When the WDT `discoverDomain` tool is used on an existing domain, a ZIP archive
 
 ### How to Build and Run
 
-**NOTE:** The image is based on a WebLogic Server image in the docker-images project: `oracle/weblogic:12.2.1.3-developer`. Build that image to your local repository before building this sample.
+**NOTE:** The image is based on a WebLogic Server image in the Oracle container registry: `container-registry.oracle.com/middleware/weblogic:12.2.1.3-dev`. You will need to login and accept the license agreement on the [Oracle Registry](https://container-registry.oracle.com/) before you can use Docker to pull the image.  
```

Comment:
Oracle container registry -> Oracle Container Registry
need to login -> need to log in
license agreement on the [Oracle Registry] -> license agreement on the [Oracle Container Registry]

Reply:
changes made as requested